include/iarc7_vision/OpticalFlowEstimator.hpp

```diff
@@ -37,6 +37,12 @@ struct OpticalFlowEstimatorSettings {
     int debug_frameskip;
     double tf_timeout;
     double max_rotational_vel;
+    std::string vector_filter;
```

Comment:
I mean I might as well fix the std::vectors while I'm at it. I don't think this will ever affect us adversely and adding the enum is just more to maintain if I want to add another filter later.

Reply:
The easy fix for the std::vectors is to just call reserve at the top of the loop to malloc the max size you might need, no need to switch to a different data structure

Reply:
As an experiment I did it but it definitely was not worth it for either.

Reply:
Oh the enum wasn't a performance thing that was a style thing

Reply:
Yes.